pkgs/misc/wine-nine-standalone/nine.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, meson
+, ninja
+, libGL
+, libdrm
+, mesa
+, xorg
+, wine
+, wine64
+}:
+
+let
+  arch = if stdenv.is32bit then "32" else "64";
+in
+stdenv.mkDerivation rec {
+  pname = "wine-nine-standalone";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "iXit";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-6mk8AqaRYEGT833o1SocJ2C9pc7wpRiQBYsI4ruS8iE=";
+  };
+
+  patches = [
+    ./0001-bootstrap-add-nixos-support.patch
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    meson
+    ninja
+    (if stdenv.is32bit then wine else wine64)
+  ];
+  buildInputs = [ libGL libdrm mesa.driversdev xorg.libX11 ];
+
+  preConfigure = ''
+    ./bootstrap.sh --distro nixos
+  '';
+
+  mesonFlags = [
+    "--buildtype" "release"
+    "--prefix" "${placeholder "out"}"
+    "--cross-file" "tools/cross-wine${arch}"
+  ];
+
+  meta = rec {
+    description = "Standalone Gallium Nine library for Wine";
+    homepage = "https://github.com/${src.owner}/${pname}";
+    changelog = "${homepage}/releases";
```

Comment:
Please use clickable links

Reply:
Thanks for the review, fixed both points.